debian/control

```diff
@@ -12,7 +12,7 @@ Build-Depends: debhelper (>= 9.20160709),
                python3-setuptools
 Build-Depends-Indep: python3-dev,
                      python3-coverage,
-                     pep8,
+                     pycodestyle,
```

Comment:
pycodestyle looks like a nice enhancement. I see two potential problems: 
1. Its only available in bionic and later so backporting u-u becomes harder.
2. Its in universe but I guess that is a problem you will deal with :)

Reply:
@mvo5 since Travis was also failing I added a fallback, this will make backporting easier.